lib/serialisers/json-0.6.js

```diff
@@ -143,8 +143,13 @@ module.exports = createClass({
       element.attributes.remove('samples');
 
       if (samples) {
-        samples = samples.map(function(sample) {
-          return sample.get(0);
```

Comment:
This would discard some sample values, for example during this case in API Blueprint: https://github.com/apiaryio/drafter/blob/v3.2.7/test/fixtures/mson/enum-sample.ast.json#L138-L159